webroot/results/p.php

```diff
@@ -1,6 +0,0 @@
-<?php
-
-if(!isset($_REQUEST['personId']) && isset($_REQUEST['i']))
-  $_REQUEST['personId'] = $_REQUEST['i'];
```

Comment:
(Not sure it's worth the effort of actually trying to fix this, however. Nginx configuration can get scary.)